instance/tests/models/test_openedx_storage_mixins.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2016 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+OpenEdXInstance Storage Mixins - Tests
+"""
+
+# Imports #####################################################################
+
+from unittest.mock import patch, call
+
+import yaml
+from django.conf import settings
+from django.test.utils import override_settings
+
+from instance.tests.base import TestCase
+from instance.tests.models.factories.openedx_appserver import make_test_appserver
+from instance.tests.models.factories.openedx_instance import OpenEdXInstanceFactory
+
+
+# Tests #######################################################################
+
+#pylint: disable=no-member
+
+class OpenEdXStorageMixinTestCase(TestCase):
+    """
+    Tests for OpenEdXStorageMixin
+    """
+    def test_ansible_s3_settings(self):
+        """
+        Test that get_storage_settings() includes S3 vars, and that they get passed on to the
+        AppServer
```

Comment:
@bradenmacdonald How does the method check "that they get passed on to the AppServer"?